docs/source/topics/calculations/usage.rst

```diff
@@ -632,17 +632,46 @@ Using the ``COPY`` mode, the target path defines another location (on the same f
    }
 
 .. note::
+    In addition to the ``COPY`` mode, the following modes, these storage efficient modes are also are available:
+    ``COMPRESS_TAR``, ``COMPRESS_TARBZ2``, ``COMPRESS_TARGZ``, ``COMPRESS_TARXZ``.
 
-   In the future, other methods for stashing may be implemented, such as placing all files in a (compressed) tarball or even stash files on tape.
+The stashed files and folders are represented by an output node that is attached to the calculation node through the label ``remote_stash``, as a ``RemoteStashFolderData`` node.
+Just like the ``remote_folder`` node, this represents a location or files on a remote machine and so is equivalent to a "symbolic link".
 
 .. important::
 
-   If the ``stash`` option namespace is defined for a calculation job, the daemon will perform the stashing operations before the files are retrieved.
+   If the ``stash`` option namespace is defined for a generic calculation job, the daemon will perform the stashing operations before the files are retrieved.
    This means that the stashing happens before the parsing of the output files (which occurs after the retrieving step), such that that the files will be stashed independent of the final exit status that the parser will assign to the calculation job.
    This may cause files to be stashed for calculations that will later be considered to have failed.
 
-The stashed files and folders are represented by an output node that is attached to the calculation node through the label ``remote_stash``, as a ``RemoteStashFolderData`` node.
-Just like the ``remote_folder`` node, this represents a location or files on a remote machine and so is equivalent to a "symbolic link".
+To avoid this scenario, you can instead, stash via a separate calculation job, for example:
+
+.. code-block:: python
+
+    from aiida.common.datastructures import StashMode
+    from aiida.orm import load_node
+
+    StashCalculation = CalculationFactory('core.stash')
+
+    calcjob_node = load_node(<CALCJOB_PK>)
+    remote_folder = calcjob_node.outputs.remote_folder
+
+    inputs = {
+        'metadata': {
+            'computer': Computer.collection.get(label="localhost"),
```

Comment:
Can we make computer optional in the `StashCalculation`? So by default the computer of the calcjob is used. Then in the docs we do not even mention it. Or we mention it in a separate section.